openmsupply/open-msupply
# 1480 program requisitions   period schedule

Fixes #1480 

# 👩🏻‍💻 What does this PR do? 
Create the `period_schedule` and `period` tables, and sync them (very mvp).

# 🧪 How has/should this change been tested? 
- Run normal tests
- Run integration tests
- Or manually by creating period schedules + periods in central to see if they appear in the database.

## 💌 Any notes for the reviewer?
Replace the `translations/store.rs` tag code with below if wanting to test manually (just a work around for all the stores being ignored if they don't have tags >.>)
```
        let tags: Vec<String> = if let Some(tags) = data.tags {
            tags.split(' ').map(|s| s.to_string()).collect()
        } else {
            Vec::new()
        };

        let store_tags = StoreTags {
            store_id: store.id.clone(),
            tags,
        };
```
